maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,193 @@
+
+
+class MaximPortkeyChatCompletions:
+    """Maxim instrumenter for Portkey chat completions."""
+
+    def __init__(self, client: Portkey, logger: Logger):
+        self._client = client
+        self._logger = logger
+
+    def create(self, *args, **kwargs):
+        """Instrumented create method that logs to Maxim."""
+        extra_headers = kwargs.get("extra_headers", None)
```

Ruff (0.11.9)

Use `kwargs.get("extra_headers")` instead of `kwargs.get("extra_headers", None)`

Replace `kwargs.get("extra_headers", None)` with `kwargs.get("extra_headers")`

(SIM910)